Sequence of Tenses/maria_ptitsyna.md

```diff
@@ -0,0 +1,19 @@
+**_6. Сергей сказал Владу, что необходимо будет решить пару организационных вопросов перед пересмотром повышения зарплаты, на что тот ему ответил, что он будет готов обсудить любые варианты._**
+
+> Sergei said Vlad, that couple of organization's answers would need to be resolved before salary revision, for what he replied, that he would be ready to discuss all options.
+
+**_7. Прежде чем приобрести новый компьютер просмотрели огромное количество обзоров и выбрали оптимальный вариант цены качества на рынке._**
+
+> Before bought new PC we had looked at a lot of reviews and chosen the optimal variant of price and quality on the market.
+
+**_8. На пером занятии я их предупредил что им придется много работать самостоятельно, и что им стоит начать читать литературу в оригинале, но они не послушали меня. Человеческую ленность никогда не побороть новым методикам обучения, они могут только её обыграть и вовлечь в процес получения знаний._**
+
+> On the first lesson I warned them, that they had to work hard by themself and they should have started to read literature in the original, but they didn't listen to me. Human laziness is never overcome by new teaching methods, they can only outgame it and engage in the process of getting knowledge. 
+
+**_9. На прошлом созвоне клиент сказал что подумает над нашим эстимейтом и коммерческим предложением, но он даже не открывал их._**
+
+> Last call the client said that he thought about our estimate and the offer, but he hasn't even opened it.
+
+**_10. Однажды Сенека сказал что ни один человек не имеет власти иметь все, что он хочет, но в его власти не хотеть того, чего у него нет, и с радостью пользоваться тем, что у него есть._**
+
+> Once Seneca said, that no one person has the power to have everything, that he wants, but in his control don't want to something, that he doesn't have, and uses with pleasure that he has.
```

Comment:
no man has the power
but he has the power not to want something he hasn't got but to to use what he has with great pleasure.